docs/templates/map-change-evidence.md

```
@@ -0,0 +1,106 @@
+pnpm sync-maps
+```
+**Result**: [ ] Pass | [ ] Fail  
+**Hash Match**: [ ] Yes | [ ] No
+
+### 2. Consistency Check
+```bash
+node scripts/verify-map-stack-consistency.mjs
+```
+**Result**: [ ] Pass | [ ] Fail  
+**Output**:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)